### PR TITLE
Matlab examples and other things

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#82 #83 #84
#### What does this implement/fix? Explain your changes.
- MATLAB examples added
- `CONTRIBUTING.md` modified
- `TEST_FILTERED_FONTS` renamed to `NON_ASCII_FONTS`
- `RANDOM_FILTERED_FONTS` and `NON_ASCII_FONTS` lists optimized
- `README.md` modified
- `appveyor.yml` modified
- `INSTALL.md` modified

### CODE REVIEW

File: MATLAB/Example1.m [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    fprintf(2,'** Python Version : %s\n',python_version);
+catch e
+    fprintf(2,'** Error : %s\n',e.message);
+end
+
+% Import art lib (version > 4.2)
+artlib = py.importlib.import_module('art');
+
+% text2art function
+% text2art(text, font=DEFAULT_FONT, chr_ignore=True)
```
Comment (sadrasabouri) [inline MATLAB/Example1.m]: add a example that use `chr_ignore` flag, it might be confusing for users?

File: MATLAB/Example1.m [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+disp(char(art9));
+
+art10 = artlib.text2art('art10','random-na');
+disp(char(art10));
+
+art11 = artlib.text2art('art11','mix');
+disp(char(art11));
+
+% tprint function
+% tprint(text, font=DEFAULT_FONT, chr_ignore=True)
```
Comment (sadrasabouri) [inline MATLAB/Example1.m]: also about `tprint` :smile:

File: MATLAB/Example1.m [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+disp('art1 message :')
+disp(char(response{'Message'}))
+
+response2 = artlib.tsave('art2','block','art.txt',true,false);
+disp('art2 message :')
+disp(char(response2{'Message'}))
+
+response3 = artlib.tsave('art3','block','art.txt',true,true,true);
+disp('art3 message :')
+disp(char(response3{'Message'}))
```
Comment (sadrasabouri) [inline MATLAB/Example1.m]: a new line is missed here

File: MATLAB/Example2.m [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+artlib.aprint('love_you',int32(1),'test');
+
+artlib.aprint('random');
+
+artlib.aprint('rand');
+
+% randart function
+% randart()
+artlib.randart();
```
Comment (sadrasabouri) [inline MATLAB/Example2.m]: a new line is missed here.

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -37,17 +37,18 @@ Please consider the following :
 	- Add font name to `UPPERCASE_FONTS`
 6. If font is Non-ASCII:
 	- Add font name to `TEST_FILTERED_FONTS` list
-	- Add font name to `RANDOM_FILTERED_FONTS` list
 	- Add a test case to `test2.py` (*Alphabetical order*)
 7. If font is ASCII:
 	- Add a test case to `test.py` (*Alphabetical order*) 
-8. Re-run `FontList.ipynb`
+8. If you want to remove this font from shuffle list: (*Optional*)
+	- Add font name to `RANDOM_FILTERED_FONTS` list
```
Comment (sadrasabouri) [inline .github/CONTRIBUTING.md]: I can't find it's logic :confused:
if we want to remove font from shuffle list we should add font to `RANDOM_FILTERED_FONTS`?
or if we want to add font to shuffle list we should do this?

**Comment by sepandhaghighi** [comment]
> I've reviewed your PR and leave some comments
> Nice Job

Thanks for your precise review 💯 
I made some changes, take a look ;-)

**Pull request merged**